### ISSUE TITLE
Toaster

### ISSUE THREAD

**Issue by halimahLDN** [Greater-London-Authority/ldn-viz-tools#760]
- [ ] Screenreader doesn't announce toaster after button click
- [ ] ~Toaster with close button doesn't autofocus once it's popped up and is difficult to tab into
https://github.com/user-attachments/assets/7eca03b7-03ff-41e1-94c6-c4c5678e8975~
- [x] ARIA dialogue and alert dialogue node has an accessible name
![image](https://github.com/user-attachments/assets/0cd5983c-4719-4be5-875f-3df2099379b6)

**Comment by halimahLDN** [comment]
According to WAI ARIA, it is not part of the [design pattern of an alert](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) to autofocus when it opens. The screen reader should announce it, though.

The errors regarding the dialog are not applicable here as the `Toast` should have a role of `alert`. Updating this removes those errors.
